jni/src/faiss_wrapper.cpp

```diff
@@ -24,6 +24,7 @@
 #include "faiss/IndexIVFPQ.h"
 
 #include <algorithm>
+#include <commons.h>
```

Comment:
"<>" usually indicates some kind of system header. Can we switch to "commons.h". See google style guide: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes